pysgrid/sgrid.py

```diff
@@ -252,6 +270,451 @@ def _save_common_components(self, nc_file):
                     dataset_grid_var.axes = ' '.join(axes)
         return grid_vars
 
+    def _get_grid_vars(self, name):
```

Comment:
In this private method you call the `grid_names` data `grid_vars`, maybe grid_vars is closer to the SGRID concept than `grid_name `.